fuel-p2p/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+edition = "2021"
+name = "fuel-p2p"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = "0.3"
+futures-timer = "3.0.2"
+ip_network = "0.4.1"
+libp2p = { version = "0.41.0" }
+log = "0.4.8"
```

Consistent version formatting:

```suggestion
futures-timer = "3.0"
ip_network = "0.4"
libp2p = "0.41"
log = "0.4"
```